tests/cli_test.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import unittest
+import sys
+
+#
+# Third party libraries
+#
+
+from mock import MagicMock, patch
+
+#
+# Internal libraries
+#
+
+from krux_cloud_health.cli import Application, main
+from krux_cloud_health.cloud_health import Interval, NAME
+from krux.stats import DummyStatsClient
+
+
+class CLItest(unittest.TestCase):
+
+    NAME = 'cloud-health-tech'
+    API_KEY = '12345'
+    INTERVAL = Interval.weekly
+    AWS_ACCOUNT = 'Krux IT'
+    COST_HISTORY_RV =  {
+            'Total': {'key': 'value'},
+            '2016-05-01': {'key': 'value'},
+            '2016-06-01': {'key': 'value'},
+            '2016-07-01': {'key': 'value'}
+        }
+    COST_CURRENT_RV = {
+            'Total': {'key': 'value'},
+            'Krux IT': {'key': 'value'},
+            'Krux Ops': {'key': 'value'}
+        }
+
+    @patch('krux_cloud_health.cli.get_logger')
+    @patch('krux_cloud_health.cli.get_cloud_health')
+    @patch('sys.argv', ['api-key', API_KEY])
+    def setUp(self, mock_get_cloud_health, mock_get_logger):
```

Comment:
Can't. The function is overriding one from the super class. They used the camel case. :/